src/TestSuite/IntegrationTestTrait.php

```diff
@@ -508,16 +498,19 @@ protected function _sendRequest($url, $method, $data = []): void
                 $this->_requestSession->write('Flash', $this->_flashMessages);
             }
             $this->_response = $response;
-        } catch (PhpUnitError $e) {
-            throw $e;
-        } catch (DatabaseException $e) {
-            throw $e;
+        } catch (\PHPUnit\Framework\Error\Error $e) {
+            $this->_exception = $e;
```

Comment:
At least PHPUnit exceptions should be rethrown. Normal applications shouldn't use PHPUnit classes.

However it could be widened to `\PHPUnit\Framework\Exception` class or `\PHPUnit\Exception` interface.

All PHPUnit asserts should immediately abort test, but not rendered with `ExceptionRenderer`. Otherwise it makes testing hard.